scapy/arch/winpcapy.py

```diff
@@ -28,7 +28,7 @@
     SOCKET = c_int
     _lib_name = find_library('pcap')
     if not _lib_name:
-      raise OSError("Cannot fine libpcap.so library")
+        raise OSError("Cannot fine libpcap.so library")
```

Comment:
I would rather keep winpcapy.py as it is. It would be easier to compare this included file to other ones.